### ISSUE TITLE
[0.8.0] Mirrow View very small

### ISSUE THREAD

**Issue by zissakos** [Nibre/MotherVR#115]
Runtime: Oculus
GTX1070

Only a very small portion of the mirror window on the desktop is used to render the game.
No matter if I change "left eye", "right eye" or both.
No matter if I change Game Settings to "Windowed", "Windowed Fullscreen" or "Fullscreen".

See this picture:
https://imgur.com/a/GRYm1TW

**Comment by Nibre** [comment]
Do you happen to have any sort of forced super-sampling turned on (DSR)?

**Comment by zissakos** [comment]
no, but reducing pixel scale in MotherVR increases the size of the rendered image.
Above post was with >2. With 1.0 I get this:
https://imgur.com/a/lYwBwEN

Would be nice if we could get one full size window on the desktop for spectators (one eye only) - no matter the pixel scale. Is that possible?
Also, does the mirror window have a performance impact? Would be nice to be able to completely turn it off.
